xled/exceptions.py

```diff
@@ -24,7 +24,7 @@ class AuthenticationError(XledException):
     """Authentication handshake wasn't successful"""
 
 
-class TokenExpiredError(XledException):
+class TokenExpiredError(ApplicationError):
```

Comment:
Why this change?

Reply:
The semantic of `ApplicationError()` is to catch issues received from payload found in HTTP body. Technically it would be application that runs on the device. `XledException()` are exceptions used to handle various issues within the `xled` package. Until now `TokenExpiredError()` wasn't raised on error caught from the device.

If there is a need to handle HTTP error 401 Forbidden I'd prefer to use separate exception that would be based on ApplicationError as you proposed here.